server/src/main/java/io/druid/client/ImmutableDruidDataSource.java

```diff
@@ -73,7 +74,6 @@ public DataSegment getSegment(String segmentIdentifier)
   @Override
   public String toString()
   {
-    // idToSegments is intentionally ignored because it is usually large
```

Comment:
But maybe rather need to keep ignore it?) If it used to be ignored for a good reason

Reply:
Good point. Added the number of segments instead of detailed segments.